modules/minimum-required-permissions-ipi-gcp.adoc

```diff
@@ -183,7 +183,8 @@ If your organization’s security policies require a more restrictive set of per
 .Required IAM permissions for installation
 [%collapsible]
 ====
-* `iam.roles.get`
```

Comment:
@brendan-daly-red-hat I think the iam.roles.get permission is still needed.

Reply:
@jianli-wei, Thanks, I re-added the iam.roles.get permission.